aiodogstatsd/client.py

```diff
@@ -272,6 +274,32 @@ def timeit(
             if not threshold_ms or value > threshold_ms:
                 self.timing(name, value=int(value), tags=tags, sample_rate=sample_rate)
 
+    def create_metric_task(
+        self,
+        coro: Awaitable[_T],
+        name: typedefs.MName,
+        *,
+        tags: Optional[typedefs.MTags] = None,
+        sample_rate: Optional[typedefs.MSampleRate] = None,
+        threshold_ms: Optional[typedefs.MValue] = None,
+    ) -> "asyncio.Task[_T]":
```

Comment:
```suggestion
    ) -> asyncio.Task[_T]:
```

Reply:
I'm pretty sure this doesn't work at runtime due to `asyncio.Task` not being a generic in 3.7/3.8

Reply:
```
tests/test_protocol.py:3: in <module>
    from aiodogstatsd import protocol, typedefs
aiodogstatsd/__init__.py:1: in <module>
    from .client import Client
aiodogstatsd/client.py:15: in <module>
    class Client:
aiodogstatsd/client.py:285: in Client
    ) -> asyncio.Task[_T]:
E   TypeError: 'type' object is not subscriptable
```